src/phoenix/trace/schemas.py

```diff
@@ -25,6 +24,9 @@ class SpanKind(Enum):
     LLM = "LLM"
     RETRIEVER = "RETRIEVER"
     EMBEDDING = "EMBEDDING"
+    PROMPT = "PROMPT"
+    PARSER = "PARSER"
```

Comment:
Do we know what these are? Should we map them to unknown for now?

Reply:
they [exist](https://github.com/langchain-ai/langsmith-sdk/blob/7ea1881198068f617b668f19aeb64fc080e2fff2/python/langsmith/schemas.py#L126-L127) in LangChain but are currently not being traced. they are supposed to be the prompt templating and output parser (e.g. relevance scores) steps